src/main/java/sandbox/UserSqlSandbox.java

```diff
@@ -171,13 +171,20 @@ public void setIndexedFixturePathBases(String fixtureName, String baseName,
     @Override
     public User getLoggedInUser() {
         if (user == null) {
-            SqliteIndexedStorageUtility<User> userStorage = getUserStorage();
-            AbstractSqlIterator<User> iterator = userStorage.iterate();
-            if (iterator.hasMore()) {
-                // should be only one user here
-                user = iterator.nextRecord();
-            } else {
-                user = null;
+            JdbcSqlStorageIterator<User> iterator = null;
+            try {
+                SqliteIndexedStorageUtility<User> userStorage = getUserStorage();
+                iterator = userStorage.iterate();
+                if (iterator.hasMore()) {
+                    // should be only one user here
+                    user = iterator.nextRecord();
+                } else {
+                    user = null;
+                }
+            } finally {
+                if (iterator != null) {
```

Comment:
Generally speaking this pattern should be

```
iterator = iterate();
try {
...
} finally {
    iterator.close();
}
```

Which is cleaner. There's no reason trying to catch errors from "userStorage.iterate();" since you can't do anything useful with the catch anyway.